public/app/features/alerting/unified/components/rule-editor/NotificationsStep.tsx

```diff
@@ -1,112 +1,263 @@
+import { css } from '@emotion/css';
 import React from 'react';
 import { useFormContext } from 'react-hook-form';
 
-import { Icon, Text, Stack } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { Icon, RadioButtonGroup, Stack, Text, useStyles2 } from '@grafana/ui';
 
 import { RuleFormType, RuleFormValues } from '../../types/rule-form';
 import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
 
 import LabelsField from './LabelsField';
 import { NeedHelpInfo } from './NeedHelpInfo';
 import { RuleEditorSection } from './RuleEditorSection';
+import { SimplifiedRouting } from './alert-rule-form/simplifiedRouting/SimplifiedRouting';
 import { NotificationPreview } from './notificaton-preview/NotificationPreview';
 
 type NotificationsStepProps = {
   alertUid?: string;
 };
+
+enum RoutingOptions {
+  'notification policy' = 'notification policy',
```

Comment:
Any particular reason the keys here are strings for an enum instead of `NotificationPolicy` etc? 🤔

Reply:
I don't remember why I used strings here 🤔, updating it 👍